packages/lab-extension/src/index.ts

```diff
@@ -121,40 +154,48 @@ const interfaceSwitcher: JupyterFrontEndPlugin<void> = {
           args: { isPalette: true },
         });
       }
-
-      switcher.addItem({ command });
     };
 
-    if (!notebookShell) {
-      addInterface({
-        command: CommandIDs.openNotebook,
-        commandLabel: trans.__('Notebook'),
-        commandDescription: trans.__('Open in %1', 'Jupyter Notebook'),
-        buttonLabel: 'openNotebook',
-        urlPrefix: `${baseUrl}tree/`,
-      });
-    }
+    switcherOptions.forEach((option) => {
+      const { command } = option;
+      addSwitcherCommand(option);
+      switcher.addItem({ command });
+    });
 
-    if (!labShell) {
-      addInterface({
-        command: CommandIDs.openLab,
-        commandLabel: trans.__('JupyterLab'),
-        commandDescription: trans.__('Open in %1', 'JupyterLab'),
-        buttonLabel: 'openLab',
-        urlPrefix: `${baseUrl}doc/tree/`,
-      });
+    let toolbarFactory: (panel: NotebookPanel) => Widget;
+    if (switcherOptions.length === 1) {
```

Comment:
Looks like this could be confusing if sometimes there is a button and other times a dropdown?

For example if a Notebook 7 user using Notebook 7 alone on a local machine, and Notebook 7 + NBClassic on Binder.

Reply:
Making the toolbar item look the same whether there is 1, 2 or more items sounds like it would provide a more consistent user experience. But that might just be personal preference.

cc @GabrielaVives who might have some input on this

Reply:
cc @isabela-pf who might also have input from UI/UX perspective

Reply:
When there is only 1 option, I think the extra friction of putting that 1 item in a dropdown isn't worth it. Also, for a give user, I wouldn't expect them to see this UI in its different variants very often, so consistency between the 1, 2, 3 item UI is less important.

Reply:
Thanks all for sharing your thoughts on this.

Then we can go with the button approach when there is only one option. We can always change later if needed based on user feedback.